samples/dashboards/demo/styling/demo.css

```diff
@@ -0,0 +1,121 @@
+@import url("https://code.highcharts.com/css/highcharts.css");
```

Comment:
![image](https://github.com/highcharts/highcharts/assets/26306846/2c21b639-ab2b-4833-9e01-f155b4dd6dad)
On my screen, the data grid does not look good. The div is a bit too small